src/inputs/DateTimeInput.tsx

```diff
@@ -190,7 +190,12 @@ class DateTimeInput extends BaseInput<DateTimeInputProps, DateTimeInputState> {
       onChange,
       ...rest
     } = this.props;
-
+    let properties = {
```

Comment:
all we need is to extract also `disableMinute` prop from `this.props` while destructuring `this.props`